accesscontroller/default.go

```diff
@@ -11,7 +10,7 @@ type Default struct {
 
 // CanAppend Checks whether a given identity can append an entry to the log.
 // This implementation allows anyone to write to the log.
-func (d *Default) CanAppend(*entry.Entry, identityprovider.Interface) error {
+func (d *Default) CanAppend(LogEntry, identityprovider.Interface, CanAppendAdditionalContext) error {
```

Comment:
Used to pass secrets or other data necessary to validate the entry, right?

Reply:
Oh no, okay, I've seen below. So the goal is to be able to access the other entries of the log to validate the current entry according to them?

Reply:
Indeed